### DISCUSSION TITLE
Pager instead of slime pager

### DISCUSSION THREAD

**Discussion by liusming** [OrchardCMS/OrchardCore#12320] [General]
Hi:

I saught there are both "Pager" and "SlimPager" both on the document https://docs.orchardcore.net/en/dev/docs/reference/modules/Navigation/#pager

However, I don't see any full function "Pager" but only "slimPager". Would you please kindly show mw the example or how-to on the "Pager" but not only "slim pager" functionality?

Thanks in advance

PS: I checked there is silmilar discussion on full pager and the answer it was not availble. However, the discussion was published in 2020. Is there any updated?

**Comment by ns8482e** [comment]
Pager is used in Admin
